src/submodules/weekly-rewards-splitting/weekly-rewards-splitting.resolver.ts

```diff
@@ -114,22 +115,24 @@ export class UserInfoByWeekSubResolver extends GenericResolver {
     async lastActiveWeekForUser(
         @Parent() parent: UserEntryFeesCollectorModel,
     ): Promise<number> {
+        const getter =
+            parent.address === scAddress.feesCollector
+                ? this.feesCollectorGetter
+                : this.farmGetter;
```

Comment:
You can extract this into a private method.

Reply:
done